.github/workflows/cmake.yml

```diff
@@ -33,6 +33,16 @@ jobs:
     - name: Install pip packages
       run: pip install -r third_party/requirements.txt
 
+    - name: Install libbacktrace
+      if: matrix.libbacktrace == '-DVAL_USE_LIBBACKTRACE_BACKTRACE=ON'
+      run: |
+        git clone https://github.com/ianlancetaylor/libbacktrace.git
+        pushd libbacktrace
```

Comment:
Are these standard gnu utilities or are integrated into sh? :D

Up to you, but I'd rather use `cd libbacktrace` and `cd ..`

Reply:
It's a bash builtin, works on Ubuntu on which these tests are run :) Although the same will be achieved with `cd` and probably it's more portable if someone would like to run this snippet in some other shell...